server.js

```diff
@@ -59,6 +59,55 @@ function startWorker(id) {
 
   const express = require('express');
   const Promise = require('bluebird');
+  const Profiler = require('./lib/tools/profiler');
+  const Uploader = require('./lib/tools/uploader');
+
+  /**
+   * Profiler setup
+   */
+
+  if (process.env.V8_PROFILER_ENABLED &&
+      process.env.AWS_ACCESS_KEY_ID &&
+      process.env.AWS_SECRET_ACCESS_KEY) {
+    logger.info('HEAP SNAPSHOT PROFILER IS ENABLED.');
+
+    const profiler = new Profiler(null, `worker #${id}`, 'heapsnapshot');
```

Comment:
Your testing instructions mention changing this `null` to a value in order to test -- should this be a config variable as well? Or could the testing instructions could have been also to change the `DEFAULT_INTERVAL_MS` in Profiler (which already suggested could be optionally defined as a config variable)